docs/DeveloperGuide.md

```diff
@@ -97,14 +97,14 @@ Given below is the Sequence Diagram for interactions within the `Logic` componen
 
 ![Structure of the Model Component](images/ModelClassDiagram.png)
 
-**API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
+**API** : [`Model.java`]
 
 The `Model`,
 
 * stores a `UserPref` object that represents the user’s preferences.
-* stores the address book data.
+* stores the Tutor Hunter data.
```

Comment:
Should this be 'Tutor Tracker'?

Reply:
Yup sorry slipped my mind our project name changed.